data/ecosystems/s/skale-network.toml

```diff
@@ -1,17 +1,78 @@
 # Ecosystem Level Information
 title = "Skale Network"
 
-sub_ecosystems = ["Owl Protocol", "Razor Network"]
+sub_ecosystems = [
+    "3Air",
+    "Bit Hotel",
+    "BlockDaemon",
+    "Blockscout",
+    "BlockWallet",
+    "Clet Name Service",
+    "CookBook",
+    "Covalent",
+    "CryptoBlades",
+    "Crypto Colosseum",
+    "Curio",
+    "DappRadar",
+    "Dirac Finance",
+    "Dirt Road Development",
+    "Dune Analytics",
+    "Enkrypt",
+    "Europa Hub",
+    "Exorde",
+    "Fireside",
+    "Hatchy Pocket",
+    "Human Protocol",
+    "Meson",
+    "MyEtherWallet",
+    "Lattice",
+    "l2beat",
+    "NFTb",
+    "NFTrade",
+    "Owl Protocol",
+    "Particle Network",
+    "Pixudi",
+    "Portis",
+    "Razor Network",
+    "ReHold",
+    "Ruby Exchange",
+    "Score Detect",
+    "Snapshot",
+    "StrayShot",
+    "Subsquid",
+    "Tank Wars Zone",
+    "The Graph",
+    "Wagmi",
+    "WalletConnect",
```

Comment:
General ecosystem

Reply:
Hey @jubos 

Tank Wars Zone - Is a dApp deployed directly onto SKALE so unsure of whether you are saying to remove them or not? 

Wagmi and Wallet Connect - both are github ecosystems that I have PRd to as well to add additional support for SKALE. I can remove them to get this PRd but would appreciate any clarification you can share on why some things can be linked and others can't.

Reply:
Resolved here: b1cff055